src/Nox.Types/Types/DateTimeDuration/DateTimeDurationTypeOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Nox.Types;
+
+public class DateTimeDurationTypeOptions : INoxTypeOptions
+{
+    public static readonly int DefaultMaxDurationDays = 365;
+    public static readonly int DefaultMaxDurationHours = 0;
+    public static readonly int DefaultMaxDurationMinutes = 0;
+    public static readonly int DefaultMaxDurationSeconds = 0;
+    public static readonly int DefaultMaxDurationMilliseconds = 0;
+
+    public static readonly int DefaultMinDurationDays = 0;
+    public static readonly int DefaultMinDurationHours = 0;
+    public static readonly int DefaultMinDurationMinutes = 0;
+    public static readonly int DefaultMinDurationSeconds = 0;
+    public static readonly int DefaultMinDurationMilliseconds = 0;
+
+    public int MaxDurationDays { get; set; } = DefaultMaxDurationDays;
+    public int MaxDurationHours { get; set; } = DefaultMaxDurationHours;
+    public int MaxDurationMinutes { get; set; } = DefaultMaxDurationMinutes;
+    public int MaxDurationSeconds { get; set; } = DefaultMaxDurationSeconds;
+    public int MaxDurationMilliseconds { get; set; } = DefaultMaxDurationMilliseconds;
+    public TimeSpan GetMaxDuration() => new TimeSpan(MaxDurationDays, MaxDurationHours, MaxDurationMinutes, MaxDurationSeconds, MaxDurationMilliseconds).Duration();
```

Comment:
TypeOptions looks good. My expectation to GetMaxDuration was to use a single field to define the max timespan, for example TimeSpan.FromMilliseconds()... From a user perspective I would argue that the expectation would be to create a MaxTimeSpan from a unit... and not set all this values, to mee seems complex..